docs/references/test-client.md

```diff
@@ -0,0 +1,373 @@
+# Test Client
+
+#### AYX SDK CLI
+
+The `ayx-sdk-cli` is the newest addition to the Alteryx Developer
+Experience product suite. It currently contains the Test Client and is
+the next iteration of CLI. The `ayx-sdk-cli` will house all the
+functionality for Desktop and future Cloud Developer Experience
+offerings.
```

Comment:
TW review:

The `ayx-sdk-cli` is the newest addition to the Alteryx Developer
Experience product suite. It currently contains the Test Client and is
the next iteration of the CLI. The `ayx-sdk-cli` houses all the
functionality for Desktop and future Cloud Developer Experience
offerings.